Source/RomanNumeral.h

```diff
@@ -13,6 +13,6 @@
  * @param output String buffer in which to store the converted Roman numeral.
  */
 void RomanNumeral_Convert(unsigned input, char *output);
-void helper(char* output, int placevalue, char* small, char* five, char* big);
+void appendLogic(char* output, int placevalue, char* small, char* five, char* big);
```

Comment:
We usually place the star next to the name of the field. That is a convention here. So this prototype would look like:
`void appendLogic(char *output, int placevalue, char *small, char *five, char *big);`

Also, if `appendLogic()` is not used by the test, or any other module on the outside, it shouldn't be listed here. It should just be in the C file as a `static` method :)